src/room.cpp

```diff
@@ -6,6 +6,7 @@
 #include "tunnler/room_message_types.h"
 
 #include "BitStream.h"
+#include "RakSleep.h"
```

Comment:
nit: RakSleep only works with win32 / pthread, we could theoretically use std threads here, but we might as well add such support in RakSleep if we find a way to maintain RakNet.